Mathlib/Analysis/SpecialFunctions/Trigonometric/Bounds.lean

```diff
@@ -53,6 +56,64 @@ theorem sin_lt {x : ℝ} (h : 0 < x) : sin x < x := by
   norm_num
 #align real.sin_lt Real.sin_lt
 
+lemma cos_quadratic_lower_bound : 1 - x ^ 2 / 2 ≤ cos x := by
+  wlog hx₀ : 0 ≤ x
+  · simpa using this $ neg_nonneg.2 $ le_of_not_le hx₀
+  suffices : MonotoneOn (fun x ↦ cos x + x ^ 2 / 2) (Ici 0)
+  · simpa using this left_mem_Ici hx₀ hx₀
```

Comment:
It is recommended to not use the focusing dot to close the goal after `suffices` (just like for `have`).
```suggestion
  suffices MonotoneOn (fun x ↦ cos x + x ^ 2 / 2) (Ici 0) by
    simpa using this left_mem_Ici hx₀ hx₀
```
Same thing below several times.